packages/client/src/components/ContactSentence.test.tsx

```diff
@@ -18,9 +18,6 @@ it("ContactSentence renders correctly without the `link` prop", () => {
   ).toBeInTheDocument();
 
   expect(queryByText("14 020", { exact: false })).toBeInTheDocument();
-  expect(
```

Comment:
Can you fix the translations in this file? `queryByText("Bel in een van deze situaties de gemeente op"`